docs/modding/suitmodding/newoutfit.md

```diff
@@ -0,0 +1,648 @@
+
+Your manifest should look like this when you've done this.
+
+![The manifest updated with localizationOverrides.](/img/suitmodding/newoutfit/localizationOverrides.png)
+
+If we deploy now this will not work, we need to supply some IDs in the manifest to make these replacements. Go to the game content tab and search for `BlackSpecialSuit`. Under the localization folder you can find two `.sweetline` files for BlackSpecialSuit, one for the name and one for the description. Click either of them, doesn't matter.
+
+In the asset overview, click the only dependency, the `LOCR` file (004B8C5124A49543). Copy this hash, and replace `Placeholder` in your manifest with `004B8C5124A49543`.
+
+Going back to the `LOCR` file's overview, click in the Preview textbox and press **CTRL + F** to open the search bar. Search for `BlackSpecialSuit`.
+
+![LOCR file search results.](/img/suitmodding/newoutfit/locr_search.png)
+
+There is the name and the description lines. But we actually can't use these as-is, we will need to hash them. First copy the whole name key, `CHAR_GLOBAL_HERO_BLACKSPECIALSUIT_M_PRO183133_NAME_A9374970-9A06-4469-993A-329FCB4DEBB4`. Open the GlacierKit text tools on the left, and in the **Localisation hash calculator**, paste the line into the text box. Copy the decimal that you get as a result. Take this to your manifest, and replace `"1"` under `localizationOverrides` with this number.
+
+Now repeat this process for the description line, replacing `"2"` with the localization decimal hash you generate in GlacierKit.
+
+When you are done, your manifest should look like this:
+
+![Manifest prepared for action.](/img/suitmodding/newoutfit/manifest_done.png)
+
+Save your manifest. Why don't we try deploying the mod in SMF and booting up the game? Give it a try, and if you've done everything right so far, your results should speak for themselves!
+
+![The new inventory image, name and description.](/img/suitmodding/newoutfit/finished_01.jpg)
+![Wearing the finished outfit.](/img/suitmodding/newoutfit/finished_02.jpg)
+
+For completion's sake, in your manifest, you should probably not stop with just English localization. If you have the means, you should translate to as many languages as you can. Intellisense will tell you which languages are valid. The more translations you provide, the bigger the potential audience of your mod, which is always cool.
+
+## Extra Credit: Peacock Addon
+
+Your mod is finished now, if your goal was just to replace a suit with your own. But we can go the extra mile too, and add our suit to the game without replacing any others. This will only work if the mod user is playing on Peacock, however. Doing this will require modifying the repository and unlockables which won't do anything if we play on IOI's servers, as we mentioned earlier.
```

Comment:
Might be worth mentioning that this also works in offline mode

**Edit:** nvm that requires the starter package thing I think

Reply:
Ah I didn't consider offline mode. Yeah that would be the purpose of the starter package patch. I already wrote it so I may as well reincorporate it slightly reworded.